api/v1alpha3/types.go

```diff
@@ -116,15 +116,6 @@ type SubnetFilter struct {
 	NotTagsAny      string `json:"notTagsAny,omitempty"`
 }
 
-// APIEndpoint represents a reachable Kubernetes API endpoint.
-type APIEndpoint struct {
-	// The hostname on which the API server is serving.
-	Host string `json:"host"`
-
-	// The port on which the API server is serving.
-	Port int `json:"port"`
-}
-
```

Comment:
Was this previously unreachable code? Trying to determine if this is intentional as it stands out a bit as unexpected in the diff.

Reply:
Yes it was unused code (except `zz_generated.deepcopy.go`)

We're using `clusterv1.APIEndpoint` from the cluster-api repo instead